src/App.tsx

```diff
@@ -1,56 +1,92 @@
-import React from 'react';
+import React, {
+  useCallback, useMemo, useState,
+} from 'react';
+import cn from 'classnames';
 import './App.scss';
+import { DropdownForm } from './components/DropdownForm/DropdownForm';
+import { DropdownMenu } from './components/DropdownMenu/DropdownMenu';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [querry, setQuerry] = useState('');
+  const [appliedQuerry, setAppliedQuerry] = useState('');
+  const [isDropdownFocus, setIsDropdownFocus] = useState(false);
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [debounceTimer] = useState(1000);
```

Comment:
The debounceTimer state does not require a useState hook as it is not being updated. It can be declared as a constant variable.